hack/libsubid_tag.sh

```diff
@@ -9,7 +9,11 @@ cc -o "$tmpdir"/libsubid_tag -l subid -x c - > /dev/null 2> /dev/null << EOF
 #include <shadow/subid.h>
 int main() {
 	struct subid_range *ranges = NULL;
+#if SUBID_ABI_MAJOR >= 4j
```

Comment:
a typo? the `j` character looks like `vi` artifact =)

Reply:
> how have you installed libsubid? I see the need for `Prog` but I don't see `shadow_logfd` defined

nevermind my last comment, I was looking at the wrong version